google-cloud-firestore/src/main/java/com/google/cloud/firestore/collection/ArraySortedMap.java

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
I think we're turning `Google Inc` into `Google LLC`

Reply:
I just merged in Master and didn't see these changes. Should we update the clients in one big PR rather than do it incrementally?

Reply:
Big PR is ok with me